### PR TITLE
PIP-32: update the consumptional fee formula

### PR DESCRIPTION
**Pull request by b00f**
This PR update the consumptional Fee Model...

### CODE REVIEW

File: PIPs/pip-31.md [hunk 1]
```diff
@@ -12,67 +12,84 @@ created: 08-08-2024
...
+However, we recommend the following configurations:
 
-### Account Creation Fee
+For most validators, we suggest setting the daily limit to `0 bytes` and the fixed fee to `0.01 PAC`.
+This configuration eliminates the consumption fee, meaning these validators will only apply the fixed fee.
 
-Each account needs to pay a `fixed_fee` the first time, after which a consumptional fee is applied.
+For validators with good resources, we recommend setting the daily limit to `280 bytes`,
+the unit price to `0.000005 PAC`, and the fixed fee to `0 PAC`.
+This allows users to send almost up to 5 free or low-fee transactions,
```
Comment (Ja7ad) [inline PIPs/pip-31.md:90]: I think 5 transactions free is high, better between 1~2 txs per day for free.

File: PIPs/pip-31.md [hunk 2]
```diff
@@ -84,48 +101,14 @@ To calculate consumption, we can define a map that associates an address with
 the total size of transactions stored in the blockchain over the last 8640 blocks.
 When a new block is committed, we update this map by iterating over all transactions in the block and
 increasing the value for each address by the size of its transactions.
-Simultaneously, the block from 8640 blocks ago would be retrieved,
+Simultaneously, the block from 8641 blocks ago would be retrieved,
```
Comment (Ja7ad) [inline PIPs/pip-31.md:104]: Why 8641 blocks?

File: PIPs/pip-31.md (hunk 1 above)
Comment (b00f) [reply PIPs/pip-31.md:90]: Based on these parameters almost 2 is free and 3 has fee less than `0.01 PAC`.

File: PIPs/pip-31.md (hunk 2 above)
Comment (b00f) [reply PIPs/pip-31.md]: Done,

**Pull request merged**